package.json

```diff
@@ -16,6 +16,15 @@
   "dependencies": {
     "ejs": "^3.1.5",
     "express": "^4.17.1",
-    "express-ejs-layouts": "^2.5.0"
-  }
+    "express-ejs-layouts": "^2.5.0",
```

Comment:
I think we should add this file and package-lock.json in .gitignore too. Otherwise if any contributor uses different versions it will come in the PR uselessly.

Reply:
Yes sir i agree and i have removed those files and added them in .gitignore for future commits